daffodil-test/src/test/resources/org/apache/daffodil/section13/text_number_props/TextNumberPropsUnparse.tdml

```diff
@@ -407,7 +407,11 @@
         <ex:e11 xmlns:ex="http://example.com">O</ex:e11>
       </tdml:dfdlInfoset>
     </tdml:infoset>
-    <tdml:document>O#########</tdml:document>
+    <tdml:warnings>
+      <tdml:warning>Schema Definition Warning</tdml:warning>
+      <tdml:warning>Explicit dfdl:length of 2 is out of range for facet maxLength='10'</tdml:warning>
```

Comment:
Suggest making these less precise like check for "explicit" "dfdl:length" "out of range" and "facet" and "maxLength='10'".

That at least gives it a chance of being a universal test that will pass for other DFDL implementations as well.